rbac/server/blockchain_transactions/role_transaction.py

```diff
@@ -17,9 +17,67 @@
 import rethinkdb as r
 from rbac.server.db.db_utils import create_connection
 from rbac.common.role.delete_role_admin import DeleteRoleAdmin
+from rbac.common.role.delete_role_owner import DeleteRoleOwner
+from rbac.common.role.delete_role_member import DeleteRoleMember
 from rbac.common.sawtooth import batcher
 
 
+async def create_delete_role_member_txns(key_pair, next_id, txn_list):
+    """Create the delete transactions for a member if user is an member of any roles."""
```

Comment:
Please add to the docstrings the details of the passed in parameters. Refer to create_delete_role_admin_txns() function as a reference.